### PR TITLE
Style Chosen doc

### PR DESCRIPTION
**Pull request by jasonlemay**
@Elrhino Let me know what you think!

### CODE REVIEW

**Comment by Elrhino** [comment]
I see there's a problem with the width of the select box being too small in the `Updating the component` dynamically and `Custom filtering sections`.

Other than that LGTM!

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/widget/EventsLogWidget.java [hunk 1]
```diff
@@ -101,6 +101,26 @@ public void onChange(ChosenChangeEvent event) {
...
+
+        @Override
+        public void onReady() {
+
+        }
+
+        @Override
+        public void onShowingDropDown() {
+
+        }
```
Comment (Elrhino) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/widget/EventsLogWidget.java]: And also please remove these overrides.

**Comment by olafleur** [comment]
Is there a reason for the image files being in that pull request? From what I can see, they are exactly the same.

**Comment by jasonlemay** [comment]
@Elrhino you can merge and fix? I'll let you handle it starting from now, do whatever please you as long as it's working and displaying right :)

@olafleur I don't know, I think Grunt never minified them (they have been dumped into the build folder) ... so I would assume now that they take slightly less space.

**Comment by Elrhino** [comment]
@jasonlemay - I'm on it.

**Pull request merged**